multi-server-adding-mailbox.adoc

```diff
@@ -0,0 +1,240 @@
+_Enter_.
+
+The installation of the mailbox server is complete.
+
+== Adding Customized Features
+
+Any customizing of themes, or Zimlets, and any signed certificates
+stored on the single-server must be added to the new mailbox
+server. See the {product-admin-guide} for information about adding the
+customized features.
+
+== Testing the Configuration
+
+To make sure that the new mail store server is correctly configured,
+create a new user on the new mailbox server and log into the account
+to verify that your configuration is correct. See
+<<_provisioning_accounts>>.
+
+== Move Mailboxes
+
+The command, `zmmboxmove`, is run to move user accounts from the mailbox
+server on the single-sever node to the new mailbox server.
+
+You can set global options to exclude items from the mailbox move. See
+the {product-admin-guide} User Accounts chapter for more information
+about the mailbox move feature.
+
+Move the following types of mailboxes:
+
+* User accounts.
+* Admin mailboxes. If you do not move the admin mailbox, you cannot
+  log into the {product-name} Web Client.
+* Spam and ham mailboxes.
+
+NOTE: If you were using _Archive and Discovery_ on the single server
+mailbox, move the archival mailboxes as well.
+
+=== Move Mailboxes Using CLI zmmboxmove
+
+. To move a mailbox to a new server
++
+----
+zmmboxmove -a <email@address> --from <servername> --to <servername>
+----
+
+. To verify that the content of the mailbox was moved successfully, go
+to the administration console, select the account that was
+moved. Click *View Mail* on the toolbar. When the account opens, verify
+that the account’s content is displayed and can be opened.
+. Purge the mailbox from the old server:
++
+----
+zmpurgeoldmbox -a <email@address> -s <oldservername>
+----
+
+== Turn Off Mailbox Server on Single-Server Node
+
+When all mailboxes have moved from the single-server node to the new
+mailbox server node, disable the Mailbox services on the original
+single-server machine.
+
+. On the original single-server node, disable the following mailbox
+server components:
++
+--
+[cols=">10%,<90%"]
+|====
+|mailbox   |`zmprov -l ms <singleserver.com> +++--+++ -zimbraServiceEnabled mailbox`
+|logger    |`zmprov -l ms <singleserver.com> +++--+++ -zimbraServiceEnabled logger`
+|stats     |`zmprov -l ms <singleserver.com> +++--+++ -zimbraServiceEnabled stats`
+|spell     |`zmprov -l ms <singleserver.com> +++--+++ -zimbraServiceEnabled spell`
+|convertd  |`zmprov -l ms <singleserver.com> +++--+++ -zimbraServiceEnabled convertd`
+|====
+
+If archiving was installed, disable it as well:
+
+----
+zmprov -l ms <singleserver.com> -- -zimbraServiceEnabled archiving
+----
+--
+
+. After the mailbox services are disabled, verify that antispam,
+antivirus, ldap, mta, snmp, proxy, and memcached are the only services
+on the original single-server node.
++
+----
+zmprov -l gs <singleserver.com> | grep -i serviceenabled
+----
+
```

Comment:
suggestion: change the example to use zmprov as it can be used (piping to grep isn't needed here)...
`zmprov -l gs <singleserver.com> zimbraserviceenabled`